src/test/compile-fail/gated-link-args.rs

```diff
@@ -9,12 +9,20 @@
 // except according to those terms.
 
 // Test that `#[link_args]` attribute is gated by `link_args`
-// feature gate.
+// feature gate, both when it occurs where expected (atop
+// `extern { }` blocks) and where unexpected.
 
 // gate-test-link_args
 
+// sidestep warning (which is correct, but misleading for
+// purposes of this test)
+#![allow(unused_attributes)]
+
+#[link_args = "-l bunny1"] //~ ERROR: the `link_args` attribute is experimental
```

Comment:
Don't forget to test it as a crate level attribute `#![link_args]` instead of as an attribute on an item.